rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -330,6 +330,14 @@ RCLCPP_PUBLIC
 std::vector<const char *>
 get_c_vector_string(const std::vector<std::string> & strings_in);
 
+/// Return the std::pair of seconds and nanoseconds from the given rcl_time_point_t.
+/**
+ * \param[in] time_point is a rcl_time_point_t
+ * \return the std::pair of seconds and nanoseconds from the rcl_time_point_t
+ */
+std::pair<int32_t, uint32_t>
+convert_rcl_time_to_sec_nanos(const rcl_time_point_t & time_point)
```

Comment:
Because utilities.hpp is also used in other files, I chose use `std::pair` instead of including `msg::Time`.